src/main/java/de/metas/ui/web/handlingunits/process/WebuiHUTransformCommand.java

```diff
@@ -133,11 +135,17 @@
 	private final List<TableRecordReference> _contextDocumentLines;
 	private final WebuiHUTransformParameters _parameters;
 
+	// Services
+	final IHandlingUnitsBL handlingUnitsBL = Services.get(IHandlingUnitsBL.class);
```

Comment:
these should be private, i guess..btw, are they actually used?

Reply:
They are no longer used. Deleted them.
Thanks for pointing out.